web/app/views/dataview/widgetSpecTable.scala.html

```diff
@@ -371,10 +374,10 @@
                 displayElements.push("Keep Undefined: " + values["keepUndefined"])
             }
 
-            var subFilterIdValue = values["subFilterId"]
+            var subFilterValue = values["subFilterTypeahead"]
 
-            if (subFilterIdValue)
-                displayElements.push("Sub Filter Id: " + subFilterIdValue)
+            if (subFilterValue)
+                displayElements.push("Sub Filter: " + subFilterValue)
 
```

Comment:
Well, this doesn't fix https://github.com/ada-discovery/ada-issues/issues/180. I don't have access to a test instance but, by looking at the code, these two lines clearly just change the display aspect (`displayElements`) of the widget - instead of the filter's id, its name is shown. As a matter of fact, this has no effect on the filter's representation (its association with a given widget), which is cleared out when a widget is edited.